README.md

```diff
@@ -4,39 +4,33 @@ A repository for [Open Data Hub](https://opendatahub.io) components Kustomize ma
 ## Community
 
 * Website: https://opendatahub.io
-* Documentation: https://opendatahub.io/docs.html (applicable for version 0.5.1)
+* Documentation: https://opendatahub.io/docs.html
 * Mailing lists: https://opendatahub.io/community.html
 * Community meetings: https://gitlab.com/opendatahub/opendatahub-community
 
-## Components
+## ODH Core Components
 
-Open Data Hub is an end-to-end AI/ML platform on top of OpenShift Container Platform which provides various tools for Data Scientists and Engineers. The components currently available are:
+Open Data Hub is an end-to-end AI/ML platform on top of OpenShift Container Platform that provides a core set of integrated components to support end end-to-end MLOps workflow for Data Scientists and Engineers. The components currently available as part of the ODH Core deployment are:
 
-* [JupyterHub](jupyterhub/README.md)
-* [Airflow](airflow/README.md)
-* [Argo Workflows](odhargo/README.md)
-* [Grafana](grafana/README.md) & [Prometheus](prometheus/README.md)
-* [Spark Operator](radanalyticsio/README.md)
-* [Kafka](kafka/README.md)
-* [Superset](superset/README.md)
-* [AI Library](ai-library/README.md)
-* [Seldon](odhseldon/README.md)
-* [Pachyderm](odhpachyderm/README.md)
-* Data Catalog
-    * [Hue](hue/README.md)
-    * [Thrift Server](thriftserver/README.md)
-* [Trino](trino/README.md)
+* [ODH Dashboard](odh-dashboard/README.md)
 * [ODH Notebook Controller](odh-notebook-controller/README.md)
 * [Data Science Pipelines](data-science-pipelines/README.md)
+* [ModelMesh](model-mesh/README.md)
+* [Grafana](grafana/README.md) & [Prometheus](prometheus/README.md)
```

Comment:
We're keeping Prometheus as a tier zero component? I thought there were discussions around driving users towards UWM.

Reply:
Do you know if the current prometheus component in ODH is still required for any monitoring workloads?

Reply:
Not off the top of my head. I know for Modelmesh we say use User Workload Monitoring.

Reply:
Thx for the reminder.  I removed it from the README since we do have plans to remove it